geoips/interfaces/yaml_based/sectors.py

```diff
@@ -13,10 +13,6 @@
 """Sector interface module."""
 
 from geoips.interfaces.base import BaseYamlPlugin, BaseYamlInterface
-from pyresample import load_area
-from geoips.image_utils.mpl_utils import create_figure_and_main_ax_and_mapobj
-from cartopy import feature as cfeature
-
```

Comment:
@jsolbrig @evrose54 Note I moved these imports inside the methods where they're used.  This seemed to take the most time of what remained for importing geoips - the registered plugins load is very fast now!

Not sure we want these inside the methods, but I don't think we want an extra several seconds for geoips to import for packages that may not need to be used.

Reply:
This is fine for now. I think we should, at some point, put some real effort into figuring out where different things should be imported in a way that balances readability, maintainability, and efficiency.